subsys/bluetooth/host/gatt.c

```diff
@@ -4122,11 +4143,20 @@ static void gatt_prepare_write_rsp(struct bt_conn *conn, uint8_t err,
 	}
 
 	len = length - sizeof(*rsp);
+
```

Comment:
Oh, I missed this one: the style we've tried to follow in Bluetooth code is "no empty line between assignment to a variable and checking the variable's value". So please don't add this line.